infra/terraform/app/providers.tf

```diff
@@ -0,0 +1,25 @@
+#Set the terraform required version
+
+terraform {
+  required_version = ">= 0.12.6"
+
+  required_providers {
+    azurerm    = "= 2.13.0"
+    databricks = "= 0.2.0"
+    helm       = "~> 1.2.2"
+    kubernetes = "~> 1.11.3"
+  }
+}
+
+provider "azurerm" {
```

Comment:
I don't think any more cloud specific content should be added.  There is already a large amount of GCP specific stuff.  We are trying to add AWS support in a way that doesn't couple us to AWS.  At some point cloud providers need to be abstracted away or isolated, so they don't permeate the dependencies.

Reply:
Again, this is to demonstrate our internal approach. There's no expectation for it to be merged.